code/game/gamemodes/cult/runes.dm

```diff
@@ -574,73 +579,43 @@ var/list/teleport_other_runes = list()
-	visible_message("<span class='warning'>[src] darkens to black and vanishes.</span>")
-	for(var/obj/effect/rune/R in orange(3,src))
-		R.visible_message("<span class='danger'>[R] fades away.</span>")
-		R.invisibility = INVISIBILITY_OBSERVER
-		R.alpha = 100 //To help ghosts distinguish hidden runes
-	for(var/mob/dead/observer/O in orange(3,src))
-		if(!O.invisibility)
-			O << "<span class='cultitalic'>You suddenly feel as if you've vanished...</span>"
-			O.invisibility = INVISIBILITY_OBSERVER
-	qdel(src)
-
-//Rite of True Sight: Turns ghosts and obscured runes visible
-/obj/effect/rune/true_sight
-	cultist_name = "Reveal Runes"
-	cultist_desc = "Reveals all invisible objects nearby, from spirits to runes."
-	invocation = "Nikt'o barada kla'atu!"
-	icon_state = "4"
-	color = rgb(255, 255, 255)
-
-/obj/effect/rune/true_sight/invoke()
-	visible_message("<span class='warning'>[src] explodes in a flash of blinding light!</span>")
-	for(var/mob/dead/observer/O in orange(3,src))
-		O << "<span class='cultitalic'>You suddenly feel very obvious...</span>"
-		O.invisibility = 0
-	for(var/obj/effect/rune/R in orange(3,src))
-		R.invisibility = 0
-		R.alpha = initial(R.alpha)
-	qdel(src)
-
- //Rite of False Truths: Makes runes appear like crayon ones
-/obj/effect/rune/make_runes_fake
-	cultist_name = "Disguise Runes"
-	cultist_desc = "Causes all nearby runes (including itself) to resemble those drawn in crayon."
-	invocation = "By'o isit!"
-	icon_state = "4"
-	color = rgb(0, 150, 0)
-
-/obj/effect/rune/make_runes_fake/invoke(mob/living/user)
-	visible_message("<span class='warning'>[src] flares brightly, then slowly dulls and appears mundane.</span>")
-	for(var/obj/effect/rune/R in range(3,src))
-		R.desc = "A rune drawn in crayon."
-
 //Rite of Disruption: Emits an EMP blast.
 /obj/effect/rune/emp
 	cultist_name = "Electromagnetic Disruption"
 	cultist_desc = "Emits a large electromagnetic pulse, hindering electronics and disabling silicons."
 	invocation = "Ta'gh fara'qha fel d'amar det!"
 	icon_state = "5"
-	color = rgb(255, 0, 0)
+	color = rgb(77, 148, 255)
 
 /obj/effect/rune/emp/invoke(mob/living/user)
+	var/turf/E = get_turf(src)
+	var/emp_strength = 0
+	for(var/mob/living/L in range(1, src))
+		if(iscultist(L))
+			var/mob/living/carbon/human/H = L
+			if(!istype(H))
+				if(istype(L, /mob/living/simple_animal/hostile/construct))
+					if(invocation)
+						L.say(invocation)
+					emp_strength++
+					continue
+			if(L.stat || (H.disabilities & MUTE) || H.silent)
+				continue
+			emp_strength++
 	visible_message("<span class='warning'>[src] glows blue for a moment before vanishing.</span>")
-	for(var/mob/living/carbon/C in orange(1,src))
-		C << "<span class='warning'>You feel a minute vibration pass through you!</span>"
-	playsound(get_turf(src), 'sound/items/Welder2.ogg', 25, 1)
-	empulse(src, 4, 8) //A bit less than an EMP grenade
-	qdel(src)
-
+	for(var/mob/living/carbon/C in range(1,src))
+		if (emp_strength < 3)
```

Comment:
This can be replaced with a switch block;
````
		switch(emp_strength)
			if(1 to 2)
				C << "<span class='warning'>You feel a minute vibration pass through you...</span>"
				playsound(E, 'sound/items/Welder2.ogg', 25, 1)
			if(3 to 6)
				C << "<span class='danger'>Your hair stands on end as a shockwave eminates from the rune!</span>"
				playsound(E, 'sound/magic/Disable_Tech.ogg', 50, 1)
			if(6 to INFINITY)
				C << "<span class=userdanger'>You chant in unison and a colossal burst of energy knocks you backward!</span>"
				playsound(E, 'sound/magic/Disable_Tech.ogg', 100, 1)
````